### ISSUE TITLE
Row ranges are not supported

### ISSUE THREAD

**Issue by igor-savin-ht** [SheetJS/js-xlsx#933]
According to Excel documentation, '1:1' is a valid range that returns entire first row. However, js-xlsx only supports ranges of format 'A1-B1'

**Comment by reviewher** [comment]
@igor-savin-ht the helper functions currently assume two-cell ranges.  Can you share a file with row or column ranges in a place that causes import errors?

**Comment by igor-savin-ht** [comment]
@reviewher Yes, that's what I mean - helper functions only support two-cell ranges. I don't think this is file-specific (I observe same behavior on all xlsx files, no matter how simple), and there are no import errors, it's just that if you pass '1:1' range, you always get empty result.

**Comment by thiennhan2310** [comment]
This libs is not supported this feature yet or this is a bug ?
